tracer/src/Datadog.Trace/Util/Http/HttpExceptionExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="HttpExceptionExtensions.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.Net.Sockets;
+
+namespace Datadog.Trace.Util.Http;
+
+internal static class HttpExceptionExtensions
```

Comment:
Should this be called `ExceptionExtensions` since the only method is on `Exception`, not `HttpException`? (level of care low)

Reply:
Yeah, _probably_ 🤔 I made it `HttpExceptionExtensions` because I wanted to put it in the `Http` sub namespace, to reduce the places it's in scope. I'll leave it there for now, but happy to shift it later